source/reference/configuration/file-setting-cli-option-mapping.txt

```diff
@@ -0,0 +1,29 @@
+.. _c2c-config-setting-option-map:
```

Comment:
do we really need this sub-page? It feels repetitive as it is already mentioned in the parent config file page